src/raygun.loader.js

```diff
@@ -280,7 +280,14 @@
 
   if (!Raygun.Utilities.isReactNative()) {
     if (document.readyState === 'complete') {
-      onLoadHandler();
+        onLoadHandler();
+    } else if (!!window.PerformanceObserver && !!window.PerformanceObserver.supportedEntryTypes && window.PerformanceObserver.supportedEntryTypes.includes('navigation')) {
```

Comment:
Optional: A possible clarity improvement here could be to split this check out into a variable so it's obvious what the containing code is for. 

Something like (var name could be improved):
 `var supportsPerformanceNavigationEntries =  !!window.PerformanceObserver && !!window.PerformanceObserver.supportedEntryTypes && window.PerformanceObserver.supportedEntryTypes.includes('navigation');
```

Then the conditional is simplified to:
```
else if (supportsPerformanceNavigationEntries)...
```